.github/workflows/changelog_verifier.yml

```diff
@@ -0,0 +1,22 @@
+name: "Changelog Verifier"
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review, labeled, unlabeled]
+
+jobs:
+  # Enforces the update of a changelog file on every pull request
+  verify-changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3.0.2
```

Comment:
How about using the floating version `v3` (and `v4` for `auto-commit-action`)? I checked their `v3` tag is in rolling updates and pointing to the latest minor version. Although there is a risk that our workflow impacted by their bug in new versions, shouldn't be a high risk.